src/components/Header/Header.tsx

```diff
@@ -1,39 +1,206 @@
-import { NavbarLarge } from './NavbarLarge';
-import { v4 as uid } from 'uuid';
-import { INavItem } from '../../models/app';
-import { NavbarSmall } from './NavbarSmall';
-import { useMediaQuery } from '@chakra-ui/react';
+import { NavbarLarge } from "./NavbarLarge";
```

Comment:
All these are not supposed to be flagged as part of your changes. Can you configure your vscode to use single quote instead of double? Otherwise, we will always have merge conflicts

Reply:
Okay